docs/userGuide/syntax/dates.md

```diff
@@ -72,28 +72,27 @@ YYYY | 2019
 
 Full formatting reference available [here](https://momentjs.com/docs/#/displaying/format/).
 
-{{ icon_example }}
 <div id="examples" class="d-none">
```

Comment:
It seems that using `div` for this element causes the example to not show up at all, which seems like expected behaviour since it's `d-none`, though this element was visible on the v3.1.1 docs. I'm not sure why `d-none` was used here though :/